src/ServerConfig.chpl

```diff
@@ -37,7 +40,70 @@ module ServerConfig
       INDEX=12,
       MULTIINDEX=13,
     };
-    
+
+    /*
+      maximum array dimensionality supported by the server
+      set by 'serverModuleGen.py' based on 'serverConfig.json'
+    */
+    config param MaxArrayDims: int = 1;
+
+    /*
+      Scalar (numpy) data types supported by the server
+      set by 'serverModuleGen.py' based on 'serverConfig.json'
+    */
+    config param  SupportsUint8 = false,
```

Comment:
I think we do support `uint8` today, I think this should default to true. We use `uint8` pdarrays for bytes of our strings. I guess this didn't show up because we have a separate casting / random gen functions for strings

Reply:
Okay that makes sense. Are you able to create an array of `uint8` in Arkouda today (e.g., `ak.ones(1000, dtype=uint8)`)?

I'm wondering because I think the `SupportsUint8` param (and the corresponding field in the config file) should indicate whether or not uint8 is a supported data type on its own. I.e., can the user directly create an array of uint8? Otherwise, someone who doesn't know that strings are implemented via uint8 may be confused if setting `uint8: false` in the config file caused strings to stop working.

And then if there was a desire to also configure whether strings are supported , there could potentially be a separate `SupportsStrings` param and a corresponding field in the config file.

Reply:
yah it's not terribly well baked in I will admit. Some methods (like `ak.ones` in ur example) are doing an explicit check for `int64` or `uint64` in the python code where this should prob just be checking if it's a numeric scalar, but there are methods that are set up to handle the smaller integer types
```python
>>> ak.arange(1000, dtype=np.uint8)
array([0 1 2 ... 229 230 231])

>>> ak.arange(1000, dtype=np.uint8).dtype
dtype('uint8')
```

Reply:
Got it, thanks! In that case I'll switch the default to `true`

Reply:
And then maybe I'll need to put more thought into this if a `SupportsStrings` config param is ever added.